hack/boilerplate/boilerplate.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python3
```

Comment:
can we make this the same from https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.py which seems the updated version of ours?